subprojects/gradle/runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/ArtifactsTestListener.kt

```diff
@@ -93,7 +93,7 @@ internal class ArtifactsTestListener(
 
             device.clearDirectory(
                 remotePath = testMetadataDirectory.toPath()
-            ).getOrThrow()
+            ).getOrElse { /* ignore errors */ }
```

Comment:
Why can we ignore cleaning?
Won't it interfere with the next test run on the device?

Reply:
You're right, i thought we somehow cleared state using snapshots, but i was wrong, `om.avito.runner.service.model.intention.State` only knows about device api level, model and installed apps

Both old and new variant are wrong, we need to remove device from pool and fail test

Reply:
it is only test metadata directory, but anyway it could lead to undesired state